apps/console/src/app/(protected)/organization-settings/billing/page.tsx

```diff
@@ -39,12 +43,20 @@ const OrganizationContent = () => {
 }
 
 const Page: React.FC = () => {
+  const { data: session } = useSession()
+  const { data: permission, isLoading } = useOrganizationRole(session)
+
   return (
     <>
-      <PageHeading heading="Billing" eyebrow="Organization Settings" />
-      <Suspense>
-        <OrganizationContent />
-      </Suspense>
+      {!isLoading && !canView(permission?.roles) && <ProtectedArea />}
+      {!isLoading && canView(permission?.roles) && (
```

Comment:
This should be `canEdit` I believe - org members should be able to see billing just the owner and admins
```suggestion
      {!isLoading && !canEdit(permission?.roles) && <ProtectedArea />}
      {!isLoading && canEdit(permission?.roles) && (
```